File: app/src/main/java/com/enhance/enhancedemo/model/Reddit.java
package com.enhance.enhancedemo.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class Reddit {

    @SerializedName("campaign")
    @Expose
    private String campaign;
    @SerializedName("launch")
    @Expose
    private Object launch;
    @SerializedName("media")
    @Expose
    private Object media;
    @SerializedName("recovery")
    @Expose
    private Object recovery;

    public String getCampaign() {
        return campaign;
    }

    public void setCampaign(String campaign) {
        this.campaign = campaign;
    }

    public Object getLaunch() {
        return launch;
    }

    public void setLaunch(Object launch) {
        this.launch = launch;
    }

    public Object getMedia() {
        return media;
    }

    public void setMedia(Object media) {
        this.media = media;
    }

    public Object getRecovery() {
        return recovery;
    }

    public void setRecovery(Object recovery) {
        this.recovery = recovery;
    }

}

File: app/src/main/java/com/enhance/enhancedemo/model/Reddit.kt
package com.enhance.enhancedemo.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class Reddit {

    @SerializedName("campaign")
    @Expose
    var campaign: String? = null
    @SerializedName("launch")
    @Expose
    var launch: Any? = null
    @SerializedName("media")
    @Expose
    var media: Any? = null
    @SerializedName("recovery")
    @Expose
    var recovery: Any? = null

}